packages/wallet/frontend/src/lib/api/paymentPointer.ts

```diff
@@ -74,6 +74,10 @@ type BasePaymentPointerArgs = {
   paymentPointerId: string
 }
 
+type PaymentPointeAssetCode = {
+  assetCode: string
+}
```

Comment:
```suggestion
type PaymentPointerAssetCode = {
  assetCode: string
}
```

I would also suggest renaming this to `PaymentPointerOP` (Payment Pointer Open Payments) and ensuring that all fields are typed, not just the asset code. Additionally, it's worth noting that the receiver account can have a different asset scale than the sender.

Reply:
Talking about this, I believe we might have a bug when a user adds an incoming payment URL as the receiver. If the incoming payment URL has a different asset, the Amount field gets populated with the amount that the incoming payment was initially created with. For example:
- Bob creates an incoming payment for 10 EUR
- Alice wants to send money to Bob from a USD account into that incoming payment
- When Alice adds the incoming payment URL as the receiver, we fetch the incoming payment details
- We populate the Amount field with the amount that was specified in the incoming payment

This will result in Alice sending Bob 10 USD, which is actually less than 10 EUR.